modules/runtime/pages/bonita-docker-installation.adoc

```diff
@@ -145,75 +145,11 @@ You can run the image with the following command:
 
 [source,bash]
 ----
-docker run --name mydbpostgres -h <hostname> -v <your-bonita-license-folder>:/opt/bonita_lic/ -d bonitasoft/bonita-postgres:12.6
+docker run --name mydbpostgres -d bonitasoft/bonita-postgres:12.6
 ----
 
 This image is built from the following https://github.com/Bonitasoft-Community/bonita-database-docker/tree/main/postgres/12[GitHub repository], which can be further adapted/customized to suit your needs.
 
-
```

Comment:
With Baptiste, we found a way to fix docker-compose, so not need to remove this.
See https://hub.docker.com/_/bonita
Moreover, the fix for the docker-compose should be done on 2021.1 or 2021.2 (I don't know exactly, the version where we removed psql client from Bonita docker image)